spec/integration/recipes/default.rb

```diff
@@ -402,6 +402,28 @@
 
 ###
 
+execute "touch -d 2016-05-02T01:23:45 /tmp/file_edit_with_content_change_updates_timestamp"
+
+file "/tmp/file_edit_with_content_change_updates_timestamp" do
+  action :edit
+  block do |content|
+    content[0 .. -1] = "Hello, world"
```

Comment:
Thanks for your response!

I fixed it using `gsub!` as `/tmp/file_edit_sample`. (8a1cf21)